### ISSUE TITLE
How to use the cookbook interactively?

### ISSUE THREAD

**Issue by eklem** [eklem/search-index-cookbook#12]
For best use of this cookbook:
```
npm install search-index-cookbook
```
Should have html files with browserified search-index javascript ready to index different data sets. Should be easy to play around in developer tool's javascript consoles. This goes for both [indexing side](https://github.com/eklem/search-index-cookbook/issues/11) and [query side](https://github.com/eklem/search-index-cookbook/issues/10) and should have example files for each recipe.

This should be possible, @fergiemcdowall ?

**Comment by eklem** [comment]
And maybe create one indexers that can take two JSON-objects: one si.add config and one si.search object?

**Comment by fergiemcdowall** [comment]
I feel like the easiest way to do it is to create the cookbook as markdown, and then allow people to read it in the markdown parser of their choice. You would also then be able to convert it really easily to html, ebook, or anything else.

**Comment by eklem** [comment]
Yes, definitely markdown on the actual documentation, but for the examples.

**Comment by eklem** [comment]
And, thinking about it twice, I'll see if I can use some of the existing example repos, or create a new one. Would be nice to be able to talk to the API in the dev JavaScript console, and watch the JSON result in the HTML part of the browser window.

**Comment by eklem** [comment]
Think I'll use this repo for indexing: https://github.com/eklem/search-index-indexer
